packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -270,12 +270,12 @@ export default defineComponent({
         .filter(Boolean)
         .join(' and ')
 
-      const usersResponse = yield clientService.graphAuthenticated.users.listUsers(
-        'displayName',
+      const usersResponse = yield clientService.graphAuthenticated.users.listUsers({
+        orderBy: new Set(['displayName']),
```

Comment:
all the `new Set(...)` are irritating me. What's the benefit over a simple array? Do we really need to enforce uniqueness?

Reply:
This is how the generated code expects it, so it's basically a question where to handle: by our abstraction layer or by the consuming party. So I guess you'd rather expect it in the abstraction layer, right?

Reply:
Yep, I'd be in favour of building the `Set`s in the abstraction layer and only expecting simple arrays as input.